src/assets/scripts/client/navigationLibrary/FixModel.js

```diff
@@ -74,23 +84,40 @@ export default class FixModel extends BaseModel {
         return this._positionModel.relativePosition;
     }
 
+    /**
+     * Facade to access spoken name
+     *
+     * @for FixModel
+     * @return {string}
+     */
+    get spoken() {
+        return this._spoken;
+    }
+
     /**
      * Lifecycle method. Should be run only once on instantiation.
      *
      * @for FixModel
      * @method init
      * @param fixName {string}
+     * @param fixSpoken {string}
      * @param fixCoordinate {array}
      * @param referencePosition {StaticPositionModel}
      * @chainable
      */
-    init(fixName, fixCoordinate, referencePosition) {
+    init(fixName, fixSpoken, fixCoordinate, referencePosition) {
         // TODO: should this be a throwing instead of returning early?
         if (!fixName || !fixCoordinate || !referencePosition) {
             return;
         }
 
         this.name = fixName.toUpperCase();
+        this._spoken = this.name; // upper-case forces spelling, experiments shows that this works best for unknown pronunciation
```

Comment:
I'm still not completely sure about this, I'll let you reviewers say what you think of this.

**Uppercase**
Forces names to always be spelled out if we don't have a defined `spoken` parameter. This will always lead to reasonable results, although we miss some opportunities where the voice engine could have pronounced the names in a better way.

**Lowercase**
This forces the voice engine to try pronouncing the names as words. Works nicely for names resembling words, such as `AUBRN`. Also works well for some hard-to-pronounce names, such as  `XDRCX`, which it spells out instead of pronouncing. Produces really weird results for other names that shouldn't be pronounced but it still tries, such as `ASDFG`, which makes it impossible to understand what it is saying.

What do you think? The lowercase version was used before, but I think it's better to force spelling now, since we can override it by defining a `spoken` instead.

Reply:
Definitely still lowercase. Most waypoints have some sort of discernable pronunciation, and that is what would be used on frequency, not spelling out a 5-letter fix. And as you mention, when the fix doesn't have a reasonable pronunciation, it spells it out. But we definitely don't want to require a spoken parameter to prevent the sim from pronouncing `APPLE` as `alpha papa papa lima echo`-- the `spoken` parameter is the exception when we want it to be pronounced a specific way which is different than what the robot lady chose :)

Reply:
Alright, I'll change it back to lowercase so we'll let the airport developer make sure they are pronounced correctly if something is weird.